public/js/pimcore/helpers.js

```diff
@@ -589,6 +589,7 @@ pimcore.helpers.showNotification = function (title, text, type, detailText, hide
             maxWidth: 350,
             closeable: true,
             align: "br",
+            autoCloseDelay: hideDelay || 3000,
```

Comment:
thanks for the info. this short delay makes the popup disappear too quick for me and it is then hard to read the message.

Reply:
what about increasing to 1 second or should i keep standard/default 3 seconds? 
It was shortened on purpose because it's not something that meant to be carefully read, but just a blinking warning, hopefully the initial problem cannot be reproduced so frequently.
I was thinking that somebody may have some weird habits (like myself 😄 ) that spams the save shortcut more than once.

Reply:
Windows 11 standard is to show notifications for 5 seconds, I think that is a sensible default.

Reply:
Resolving, keeping the same default delay.

I've just noticed that the `hideDelay` is meant for the delay before hiding after user mouseover event on the toast notification, and not related to the autocloseDelay, so it would be a wrong fix.